example/simple_testing/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
nit: still don't want the simple_testing crate unless we absolutely have to

Reply:
oh yeah, i'll remove this! i just re-included it when this PR was just a manual example.

Reply:
done!